src/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.nav {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  align-items: center;
+  color: black;
+  margin-top: 50px;
```

Comment:
- [ ] try to reduce the margin-top value and instead of using px try using rem or em